code/modules/mob/living/carbon/alien/damage_procs.dm

```diff
@@ -1,21 +1,26 @@
 
+/mob/living/carbon/alien/apply_damage(damage = 0,damagetype = BRUTE, def_zone = null, blocked = 0)
+	var/hit_percent = (100-blocked)/100
+	if(!damage || (hit_percent <= 0))
+		return 0
+	switch(damagetype)
+		if(BRUTE)
+			adjustBruteLoss(damage * hit_percent)
+		if(BURN)
+			adjustFireLoss(damage * 2 * hit_percent) // Xenos take double burn damage, MUY IMPORTANTE
```

Comment:
Oh, ugh, this is not the proper fix; you need to override alien limbs, like `/obj/item/bodypart/r_arm/alien`, to take double burn; doing this will mean xenos won't take double damage from fire/hot air.